### PR TITLE
Add missing fields to billing admin transaction class

### PR DESCRIPTION
**Pull request by justinvdm**
We aren't currently displaying the new fields added in #1118 in the admin UI.

### CODE REVIEW

**Comment by justinvdm** [comment]
Ready for review.

**Comment by justinvdm** [comment]
Ready for review.

**Comment by hodgestar** [comment]
:+1:

File: go/billing/models.py [hunk 1]
```diff
@@ -183,7 +183,7 @@ def message_credit_cost(self):
     def storage_credit_cost(self):
         """Return the calculated cost per session (in credits)."""
         return self.calculate_storage_credit_cost(
-            self.session_cost, self.markup_percent)
+            self.storage_cost, self.markup_percent)
```
Comment (hodgestar) [inline go/billing/models.py:186]: Hmm. Was this a bug we missed?

Comment (hodgestar) [inline go/billing/models.py:186]: I think I saw this earlier but maybe it got lost in all the merging somehow?

Comment (justinvdm) [inline go/billing/models.py:186]: I remember this too. Possible reasons I can think of:
- it was actually the `calculate_storage_credit_cost` line that was noticed in an earlier PR
- the fix got lost in the merge (I hope its not that, could mean other things got lost)
- I forgot to fix it before merging the relevant PR into this PR

**Pull request merged**